app/models/ticket_purchase.rb

```diff
@@ -71,6 +74,12 @@ def pay(payment)
     end
     Mailbot.ticket_confirmation_mail(self).deliver_later
   end
+
+  def one_registration_ticket_per_user
+    if ticket.try(:registration_ticket?) && quantity > 1
+      errors.add(:quantity, 'cannot be greater than one for registration tickets.')
```

Comment:
:art: maybe _can only be one_ instead of _greater than one_

Reply:
👍